public/lessons/server_side_javascript/parameters_in_express.md

```diff
@@ -1,60 +0,0 @@
-# Parameters in Express
```

Comment:
@Habenzy Are these being moved somewhere else? This content is quite useful for students in the later half of the course.

Reply:
@Habenzy I'm going to accept this, but I want to review these lessons later this week or next.